File: tests/language_completion/fixtures/cpp/boolean_logic.cpp
#include <iostream>
#include <string>

int main()
{
    int total = 8;
    bool ready = true;
    std::string label = "blocked";
    if (ready && total == 8)
    {
        label = "ready";
    }
    std::cout << "cpp_bool " << label << std::endl;
}

File: tests/language_completion/fixtures/csharp/boolean_logic.cs
using System;

public class Program
{
    public static void Main()
    {
        int total = 8;
        bool ready = true;
        string label = "blocked";
        if (ready && total == 8)
        {
            label = "ready";
        }
        Console.WriteLine("cs_bool " + label);
    }
}
